src/WebJobs.Script.WebHost/Metrics/HostMetricsProvider.cs

```diff
@@ -19,21 +20,24 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.Metrics
 {
     public class HostMetricsProvider : IHostMetricsProvider, IDisposable
     {
+        private const string HeartbeatMetricName = "azure.functions.host_metrics_heartbeat";
```

Comment:
Recommend adding this to the existing MetricEventNames class. Also, looking at the naming convention for the rest of the metrics, I don't see _ being used

Reply:
_ is okay for metric names, we do that for the other host metrics i.e.

https://github.com/Azure/azure-functions-host/blob/2557cf38e1de490f4ea423492ea2ec23fb78f24a/src/WebJobs.Script/Metrics/HostMetrics.cs#L23

I went with `azure.functions.host_metrics_heartbeat` instead of `azure.functions.heartbeat` to avoid taking up a name that might be utilized by other health check work coming up, but perhaps I am over thinking this. @RohitRanjanMS @jviau fyi

Reply:
Yes, `_` is the correct separator to use for multi-word names